cspell/project-words.txt

```diff
@@ -0,0 +1,111 @@
+OSRD
+VecDeque
+Odoo
+Motis
+möglich
+Remarquable
+opensource
+TVDS
+autonumber
+mgsin
+Isort
+pytype
+stringifying
+mtlynch
+pytest
+parallelizable
+macbooks
+virtiofs
+osxkeychain
+creds
+relref
+balise
+odometry
+Tdriver
+Tindication
+FLOI
+Twarning
+maxmemory
+allkeys
+exploitability
+EPSG
+autosquash
+MRSP
+Dreiländer
+DGEX
+QGIS
+Kutta's
+Kutta
```

Comment:
[Kutta](https://fr.wikipedia.org/wiki/Martin_Wilhelm_Kutta) should be in `names.txt` I would say, what do you think?